style.css

```diff
@@ -1,245 +1,328 @@
+    margin: 0;
+    padding: 0.5em;
+    display: flex;
+  }
 
-.links li {
-  margin-right: 20px;
-}
+  .links li {
+    margin-right: 20px;
+  }
 
-a {
-  color: #6071fe;
-  text-decoration: none;
-}
+  a {
+    color: #6071fe;
+    text-decoration: none;
+  }
 
-.hamburger {
-  display: none;
-}
+  .hamburger {
+    display: none;
+  }
 
-.header {
-  padding-left: 300px;
-  padding-right: 300px;
-  padding-top: 200px;
-  height: 100vh;
-  border-radius: 0 0 0 100px;
-  background: white;
-  background-size: contain;
-  background-image: url("images/header.png");
-}
+  .header {
+    padding-left: 300px;
+    padding-right: 300px;
+    padding-top: 200px;
+    height: 100vh;
+    border-radius: 0 0 0 100px;
+    background: white;
+    background-size: contain;
+    background-image: url("images/header.png");
+  }
 
-.headline {
-  font-weight: 600;
-  font-size: 4em;
-}
+  .headline {
+    font-weight: 600;
+    font-size: 4em;
+  }
 
-.headline-description {
-  font-weight: 100;
-  font-size: 2em;
-  padding-top: 10px;
-  padding-bottom: 10px;
-}
+  .headline-description {
+    font-weight: 100;
+    font-size: 2em;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
 
-.connect {
-  font-weight: 500;
-  padding-top: 20px;
-  padding-bottom: 20px;
-  font-size: 1.2em;
-  color: #6071fe;
-  text-transform: uppercase;
-}
+  .connect {
+    font-weight: 500;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    font-size: 1.2em;
+    color: #6071fe;
+    text-transform: uppercase;
+  }
 
-.social-links {
-  display: flex;
-  width: 20%;
-  justify-content: space-between;
-  align-items: center;
-  margin-top: 20px;
-  margin-bottom: 40px;
-}
+  .social-links {
+    display: flex;
+    width: 20%;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 20px;
+    margin-bottom: 40px;
+  }
 
-.works {
-  display: grid;
-  grid-template-columns: 3;
-  gap: 142px;
-  align-items: center;
-  background-color: #f7f7f7;
-  justify-content: center;
-  padding-top: 100px;
-  padding-bottom: 100px;
-}
+  .works {
+    display: grid;
+    grid-template-columns: 3;
+    gap: 142px;
+    align-items: center;
+    background-color: #f7f7f7;
+    justify-content: center;
+    padding-top: 100px;
+    padding-bottom: 100px;
+  }
 
-.work {
-  padding: 24px;
-  border: gray solid 0.3px;
-  border-radius: 20px;
-  background: white;
-  display: flex;
-}
+  .work {
+    padding: 24px;
+    border: gray solid 0.3px;
+    border-radius: 20px;
+    background: white;
+    display: flex;
+  }
 
-.work-subtitle {
-  display: flex;
-  gap: 12px;
-  align-items: center;
-}
+  .work-subtitle {
+    display: flex;
+    gap: 12px;
+    align-items: center;
+  }
 
-.work-title {
-  font-weight: 600;
-  font-size: 3em;
-  color: black;
-}
+  .work-title {
+    font-weight: 600;
+    font-size: 3em;
+    color: black;
+  }
 
-.work-details {
-  align-items: center;
-  padding-left: 46px;
-  padding-right: 46px;
-}
+  .work-details {
```

Comment:
- [ ] I kindly suggest that you make sure that your project feature and project description switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).

<img width="504" alt="image" src="https://user-images.githubusercontent.com/52934128/186461532-9cf75e7e-83c7-49d2-9fdd-18a063ae0e15.png">